ray_lightning/ray_ddp.py

```diff
@@ -77,19 +79,24 @@ class RayAccelerator(DDPSpawnAccelerator):
     def __init__(self,
                  num_workers: int = 1,
                  num_cpus_per_worker: int = 1,
-                 use_gpu: bool = False):
+                 use_gpu: bool = False,
+                 init_hook: Callable = None):
         super().__init__(trainer=None, nprocs=0)
         self.nickname = "ddp_ray"
         self.num_workers = num_workers
         self.num_cpus_per_worker = num_cpus_per_worker
         self.use_gpu = use_gpu
         self.workers = []
+        self.init_hook = init_hook
 
     def _create_worker(self):
         """Creates Ray actor."""
-        return RayExecutor.options(
+        worker = RayExecutor.options(
             num_cpus=self.num_cpus_per_worker,
             num_gpus=int(self.use_gpu)).remote()
+        if self.init_hook:
+            worker.execute.remote(self.init_hook)
```

Comment:
Don't we need to `ray.get` here?
```suggestion
            ray.get(worker.execute.remote(self.init_hook))
```

Reply:
the reason I didn't was because this makes worker creation serial, but it's not a big deal/the new approach may catch errors faster

Reply:
Gotcha, yeah. I'll modify it to call ray.get just once at the end for all the workers.